### PR TITLE
CA-593 Add a SwaggerUI page to Bond

### PR DESCRIPTION
**Pull request by wchamber**
Adds an OpenApi v3 yaml describing bonds endpoints.
Make Bond host the SwaggerUI page with flask-swagger-ui.
Add an automated test to check that the SwaggerUI page exists.

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary. Documentation PRs only need 1 thumb.
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by wchamber** [comment]
Note that the 404 responses are assuming pull #122 is submitted.

File: swagger/api-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                type: string
+
+    LinkNotFound:
+      description: Unable to find a link for the user to the provider. Consider re-linking with an oauthcode.
+
+  securitySchemes:
+    googleAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
```
Comment (gpolumbo-broad) [inline swagger/api-docs.yaml]: I do not think Google Access Tokens are JWT

**Review by gpolumbo-broad — APPROVED** [review]
What do we know about how authentication works or doesn't work?  Will users be able to log in using the swagger UI?  Or will they need to BYO Bearer token?

**Comment by wchamber** [comment]
> What do we know about how authentication works or doesn't work? Will users be able to log in using the swagger UI? Or will they need to BYO Bearer token?

The swagger UI will ask them to add their bearer token, which they need to generate themselves.

Screenshot from 2020-02-18 09-34-54

File: swagger/api-docs.yaml (hunk 1 above)
Comment (wchamber) [reply swagger/api-docs.yaml]: done

**Review by dvoet — APPROVED** [review]
awesome!

**Pull request merged**